dll/win32/shell32/lang/pt-PT.rc

```diff
@@ -182,8 +182,8 @@ BEGIN
     LTEXT "", -1, 8, 31, 223, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Tipo de destino:", IDC_SHORTCUT_TYPE, 8, 38, 68, 10
     EDITTEXT IDC_SHORTCUT_TYPE_EDIT, 79, 37, 150, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "Localização do destino:", IDC_SHORTCUT_LOCATION, 8, 55, 68, 10
-    EDITTEXT IDC_SHORTCUT_LOCATION_EDIT, 79, 53, 150, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
+    LTEXT "Localização de destino:", IDC_SHORTCUT_LOCATION, 8, 50, 68, 15
```

Comment:
Most of LTEXTs have a width of 10, except those that you have changed here (and below): why is it so?

Reply:
these texts occupy two lines. So I had to increase the height

Reply:
![Before](https://user-images.githubusercontent.com/49964946/93545522-87dbf900-f958-11ea-82a4-cfe6a5b5a507.png)
We can't see all the tet

Reply:
![After](https://user-images.githubusercontent.com/49964946/93578574-63ead880-f995-11ea-8376-f5d648b782d9.png)
Aso in"Tipo de ficheiro"

Reply:
ok thanks @zecarlos1957 I understand now :) But note, in the last screenshot, that there appears to be an alignment problem of the text zone "2,00 KB" with the other ones above it (the "189 bytes" and the "X:\Profiles\...").

Also, the livecd being ISO, i don't remember whether this stores any "file accessed date" value (if not, it defaults to "0" which means that 1st january 1601).

Reply:
In the pt-PT.rc file all the edit boxes are aligned.
It seems that the problem lies in the file <drvdefext.cpp> by a space.
I will investigate and open a jira tiket

Reply:
my mistake. My apologies. Found it